samples/dymaptic.GeoBlazor.Core.Sample.Shared/Pages/LocationMethods.razor

```diff
@@ -0,0 +1,115 @@
+﻿@page "/locator-methods"
+@inject Locator Location
+<PageTitle>Locator Methods</PageTitle>
+<h1>Locator Methods</h1>
+
+
+<div class="form-group" style="border: 1px solid black; padding: 0.5rem;">
+
+    <h2>Addresses To Locations method</h2>
+    <code>var locations =  _locations2 = await Location.AddressesToLocations([new Address("380 New York Street", "Redlands", "CA", "92373"), new Address("400 New York Street", "Redlands", "CA", "92373")]);</code>
+    <br />
+    <br />
+    <label>
+
+        <button @onclick="AddressesToLocations">Addresses To Locations</button>
+        <br />
+        <b>Address Candidates:</b>
+        @foreach (var location in _locations2)
+        {
+            <div>@location?.Address  @location?.Score</div>
+            <br />
+        }
+    </label>
+</div>
+<div class="form-group" style="border: 1px solid black; padding: 0.5rem;">
+
+    <h2>Address To Locations method</h2>
+    <code>var locations = await Location.AddressToLocations(new Address("380 New York Street", "Redlands", "CA", "92373"), outFields:new(){"*"});</code>
```

Comment:
i think we are missing `_locations1` here.

Reply:
this is just the example, so it can be simple locations